NAMESPACE

```diff
@@ -7,13 +7,21 @@ export(ReadData_AgProd)
 export(ReadData_LN3_LandLeaf)
 export(SCENARIO.INFO)
 export(ScenarioInfo)
+export(calc_loglikelihood)
+export(calc_post)
+export(calc_prior)
 export(get_hindcast_AgProdChange)
 export(get_hindcast_prices)
+export(get_historical_land_data)
```

Comment:
At some point, I should have probably developed a style guide, but I guess we can just let it go now?

Reply:
I'm not sure what you're referring to here.  Is it that "historical" should be "hindcast"?

Reply:
Sorry, this wasn't related to this line in particular. I just noticed at this point, that I'm inconsistent with my use of capital letters and underscores.